libappimage/src/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+###
+# AppImage deployment utils package
+###
+
+include(../cmake/cmake_import_manual_targets.cmake)
+
+find_package(PkgConfig)
+
+pkg_check_modules(GLIB REQUIRED glib-2.0)
+include_directories(${GLIB_INCLUDE_DIRS})
+
+pkg_check_modules(GIO REQUIRED gio-2.0)
+include_directories(${GIO_INCLUDE_DIRS})
+
+pkg_check_modules(CAIRO REQUIRED cairo)
+include_directories(${CAIRO_INCLUDE_DIRS})
+
+
+include_directories("../../squashfuse/")
+include_directories("../../libarchive-3.3.1/libarchive")
+
+SET(LIBAPPIMAGE_DEPLOYMENT_SRC
+    appimage.h
+
+    ../../shared.c
+    ../../getsection.c
+    ../../notify.c
+    ../../elf.c
+)
+
+add_library(appimage SHARED ${LIBAPPIMAGE_DEPLOYMENT_SRC})
```

Comment:
I know that CMake does add a `lib` prefix by default, hence most people create targets called like this, but I like having the `lib` prefix in the target name as well.

See https://github.com/AppImage/AppImageUpdate/blob/rewrite/src/CMakeLists.txt#L21-L22 for how to "fix" CMake's behavior.